pilates/activitysim/preprocessor.py

```diff
@@ -697,22 +753,46 @@ def _create_land_use_table(
 
 
 def _get_zones_table(
-        datastore, region, asim_zone_id_col='TAZ',
-        default_zone_id_col='zone_id'):
+        datastore, region, state_fips, county_codes, zonification, 
```

Comment:
again, maybe `zone_type` instead of `zonification`? It's not immediately clear what `zonification` means even though its kind of fun.